cmd/util/ledger/migrations/account_based_migration.go

```diff
@@ -315,7 +315,27 @@ func MigrateGroupConcurrently(
 	return migrated, nil
 }
 
-var knownProblematicAccounts = map[common.Address]string{}
+var knownProblematicAccounts = map[common.Address]string{
```

Comment:
`knownProblematicAccounts` is already used in `MigrateGroupConcurrently` to skip whole accounts that are broken. We should maybe removed that logic, and then rename `knownProblematicAccounts` here so that is only used for `FixBrokenReferencesInSlabsMigration`, e.g. `fixBrokenSlabReferencesTestnetAccounts`.

Maybe also add an `init` function and check at startup of the program, that all addresses in the set are Testnet addresses (can use `flow.Testnet.Chain().IsValid()` for that)

Reply:
Good catch!  Done in commit aa5893467708bb578675d3e0fba827ba1088755d.